atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1345,11 +1345,19 @@ bool WebContents::IsOffScreen() const {
 
 void WebContents::OnPaint(const gfx::Rect& dirty_rect,
                           const gfx::Size& bitmap_size,
+                          const int pixel_size,
                           void* bitmap_pixels) {
   v8::MaybeLocal<v8::Object> buffer = node::Buffer::New(
```

Comment:
Calling `node::Buffer::New` means the data is transferred to Node to manage, so once the returned `data` is garbage collected the `bitmap_pixels` would be deleted, while it is still being managed by someone class. I think we should do `node::Buffer::Copy` here.